x-pack/plugins/lists/server/services/exception_lists/utils/index.test.ts

```diff
@@ -9,7 +9,7 @@ import sinon from 'sinon';
 import moment from 'moment';
 import uuid from 'uuid';
 
-import { transformCreateCommentsToComments, transformUpdateCommentsToComments } from './utils';
+import { transformCreateCommentsToComments, transformUpdateCommentsToComments } from './';
```

Comment:
You don't really want imports to end with a slash such as `./` and instead use `.` because sometimes there is a regular expression within jest or somewhere else we are looking for `.` ... `./` is kind of non-standard.